src/traces/contour/colorbar.js

```diff
@@ -52,7 +53,7 @@ module.exports = function colorbar(gd, cd) {
         })
         .levels({
             start: contours.start,
-            end: contours.end,
+            end: endPlus(contours),
```

Comment:
`colorbar` actually adds some more to `end` for the same purpose - but it turns out not to matter in this case, additional expansion doesn't have any ill effects that I could find, as long as it always ends up at or beyond the level we used to make the `colorscale`. Sometime though we may want to fold ALL the places we extend a range a little for rounding purposes into the same `endPlus` mold.